resources/howtos/search_bar_for_docs.md

```diff
@@ -0,0 +1,201 @@
+# Integrate a relevant search bar to your documentation
+
+You might have noticed the search bar in this documentation.
+
+![MeiliSearch docs search bar demo](/tuto-searchbar-for-docs/vuepress-searchbar-demo.gif)
+
+And you are probably wanting the same for your own documentation!
+
+This tutorial aims to explain in detail the required steps to set up a relevant and powerful search bar in your documentation with MeiliSearch 🚀
+
+1. [Run a MeiliSearch Instance](#run-a-meilisearch-instance)
+2. [Scrap your content](#scrap-your-content)
+3. [Integrate the Search Bar](#integrate-the-search-bar)
+
+## Run a MeiliSearch Instance
+
+First of all, you need your documentation content to be scraped and pushed into a MeiliSearch instance.
+
+This can be done in a few steps. MeiliSearch is open-source and can run on your own server!
+
+Check out this well-explained tutorial about [how to run MeiliSearch in production](/resources/tutorials/running_production.md).
+If you are used to DigitalOcean droplets, you can easily [create one](https://marketplace.digitalocean.com/apps/meilisearch?action=deploy&refcode=7c67bd97e101).
+
+::: note
+
+The host URL and the API key you will provide in the next steps correspond to the credentials of this MeiliSearch instance.
+
+:::
+
+## Scrap your Content
+
+The Meili team provides and maintains a [scraper tool](https://github.com/meilisearch/docs-scraper) to scrap your website content and automatically index it into MeiliSearch.
+
+### Configuration File
+
+The scraper needs a configuration file to know the content to be scraped thanks to the selectors (e.g. the HTML tag).
+
+Here is an example of a basic configuration file:
+
+```json
+{
+  "index_uid": "docs",
+  "start_urls": ["https://www.example.com/doc/"],
+  "sitemap_urls": ["https://www.example.com/sitemap.xml"],
+  "stop_urls": [],
+  "selectors": {
+    "lvl0": {
+      "selector": ".docs-lvl0",
+      "global": true,
+      "default_value": "Documentation"
+    },
+    "lvl1": {
+      "selector": ".docs-lvl1",
+      "global": true,
+      "default_value": "Chapter"
+    },
+    "lvl2": ".docs-content .docs-lvl2",
+    "lvl3": ".docs-content .docs-lvl3",
+    "lvl4": ".docs-content .docs-lvl4",
+    "lvl5": ".docs-content .docs-lvl5",
+    "lvl6": ".docs-content .docs-lvl6",
+    "text": ".docs-content p, .docs-content li"
+  }
+}
+```
+
+The `index_uid` field is the index identifier in your MeiliSearch instance in which your website content is stored. The scraping tool will create a new index if it does not exist.
+
+The `docs-content` class is the main container of the textual content in this example. Most of the time, this tag is a `<main>` or an `<article>` HTML element.
```

Comment:
This is from their official doc
> Add a static class DocSearch-content to the main container of your textual content. Most of the time, this tag is a <main> or an <article> HTML element.

Which is different to what you are saying.

> The `docs-content` class is the main container of the textual content in this example. Most of the time, this tag is a `<main>` or an `<article>` HTML element.


Add a static class DocSearch-content to the main container of your textual content. Most of the time, this tag is a <main> or an <article> HTML element.